.github/workflows/wheels-dependencies.sh

```diff
@@ -137,14 +137,16 @@ function build_libavif {
         && CMAKE_POLICY_VERSION_MINIMUM=3.5 cmake \
             -DCMAKE_INSTALL_PREFIX=$BUILD_PREFIX \
             -DCMAKE_INSTALL_LIBDIR=$BUILD_PREFIX/lib \
+            -DCMAKE_INSTALL_NAME_DIR=$BUILD_PREFIX/lib \
+            -DCMAKE_MACOSX_RPATH=OFF \
             -DCMAKE_BUILD_TYPE=Release \
-            -DBUILD_SHARED_LIBS=OFF \
+            -DBUILD_SHARED_LIBS=ON \
             -DAVIF_LIBSHARPYUV=LOCAL \
             -DAVIF_LIBYUV=LOCAL \
             -DAVIF_CODEC_AOM=LOCAL \
+            -DCONFIG_AV1_DECODER=0 \
```

Comment:
IMPORTANT: The `-DCONFIG_AV1_DECODER=0` needs to be passed to libaom's cmake command. We are passing it to libavif's cmake command here. Does libavif's cmake command forward it to libaom's cmake command? Otherwise this has no effects and should be removed.

Does this require changing libavif's cmake/Modules/LocalAom.cmake?

Reply:
To your other question though, it would probably make sense for us to `set(CONFIG_AV1_DECODER 0)` in `LocalAom.cmake` if `AVIF_CODEC_AOM_DECODE` is `OFF`, so that passing this is unnecessary.

Reply:
Frankie: Thanks for the explanation. Let's add a comment to note that `CONFIG_AV1_DECODER` is a cmake flag of libaom.

I will take a stab at modifying libavif's cmake/Modules/LocalAom.cmake.